test/expressions-test.js

```diff
@@ -93,13 +140,16 @@ describe('Expressions', () => {
         })).to.equal('http://example.com/api/v1');
       });
 
-      it('inserts nothing if variable is found but undefined', () => {
```

Comment:
what happens in this case now? do we resolve it to the string "undefined"? 🤔

Reply:
prints `undefined`